src/hooks/use_react_window.ts

```diff
@@ -22,11 +22,19 @@ export const useList = () => {
     };
   };
 
+  const resetToTop = () => {
+    // this reset list to the first row when click a new tab
```

Comment:
Actually, It's just a util function. It doesn't care it there is a new tab or not.
If you want to make it informative you should go the following:
https://stackoverflow.com/questions/23072286/where-is-the-syntax-for-typescript-comments-documented